inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,23 @@ def create_record(record):
     return json
 
 
+def update_documents_and_figures_for_record(record):
+    for attachment in chain(record.get('documents', []), record.get('figures', [])):
+        file_url = attachment.get('url')
+        filename = '{}_{}'.format(record.get('recid') + attachment.get('key'))
```

Comment:
Something's off here! And probably an assertion is missing from the tests.

Reply:
We discussed to have the recid attached to the filename for making filenames unique. This is called inside `record_insert_or_replace` which actually ensures that there is a recid. For the tests I refactor the assertions and the snippets.

Reply:
oups now I saw it :P I am fixing it!